Sources/FoundationNetworking/URLSession/Message.swift

```diff
@@ -111,6 +111,10 @@ struct _Delimiters {
     static let Space = UnicodeScalar(0x20)
     static let HorizontalTab = UnicodeScalar(0x09)
     static let Colon = UnicodeScalar(0x3a)
+    static let Backslash = UnicodeScalar(0x5c)!
+    static let Comma = UnicodeScalar(0x2c)!
+    static let Dquote = UnicodeScalar(0x22)!
```

Comment:
Nit: Mind if we call this `DoubleQuote`? We already spell HTAB `HorizontalTab` instead of `Htab`, and it just reads a little more clearly, I think, in the context where we have comments referring to "dequoted" strings and then code that refers to "dquote".

Reply:
Sure! Anything to reduce confusion. Change pushed.